DayrSuryan/DSEthiop7.xml

```diff
@@ -361,7 +361,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         <item xml:id="a1">
                            <locus target="#iv"/>
                            <desc type="OwnershipNote">Ownership note stating that the manuscript belongs to
-                              <persName ref="PRS12557Barha"><roleName type="title"><hi rendition="simple:italic">Ḥalaqā</hi></roleName> Barha</persName>.
+                              <persName ref="PRS12557Barha"><roleName type="title"><hi rendition="simple:italic">Ḫalaqā</hi></roleName> Barha</persName>.
```

Comment:
If we transliterate, the name of the brave Ḫalaqā should be spelled Barhe.

Reply:
I suggest that we remove (!) after his name. በርሄ is not a deviation, it is the old Tigrigna orthography

Reply:
but it is በሬሄ here

Reply:
I see, it interfered with PRS12557Barh**a**

Reply:
I think the ID doesn't matter, but the sic refers to ሬ